mapflow/functional/layer_utils.py

```diff
@@ -279,6 +280,48 @@ def add_layer(self, layer: Optional[QgsMapLayer]) -> None:
         else:  # assume user opted to not use a group, add layers as usual
             self.project.addMapLayer(layer)
 
+    def add_layer_by_order(self, layer: Optional[QgsMapLayer], order) -> None:
+        if not layer:
+            return
+        self.layer_group = self.layer_tree_root.findGroup(self.settings.value('layerGroup'))
+        if self.add_layers_to_group:
```

Comment:
I believe this code is used elswhere in layer management, shouldn't we wrap this in a function and reuse?

Reply:
Yes, I copy-pasted that from another function and slightly edited it. Don't now why I did it, because I could just edit the existing function and define a default value of an argument I needed to add (instead of a constant value), so nothing breaks in other places where this function is used. If you were talking about that - I fixed it.